docs/guide/custom-login-requirements.md

```diff
@@ -130,12 +226,18 @@ To complete the requirement we also need a Vue component on the browser side. As
 </AposCodeBlock>
 
 ::: note
-Don't forget to add `APOS_DEV=1` at the start of your `npm run dev` command if you are actively developing Apostrophe admin UI components. Otherwise Apostrophe will not rebuild the admin UI every time.
+Don't forget to set the `APOS_DEV` environment variable to `1` when developing admin UI code. Otherwise Apostrophe will not rebuild the admin UI every time.
+
+If your project is derived from `a3-boilerplate` you can type:
```

Comment:
This could be clearer regarding what the script does and where you type it. It's possibly the first time the developer has run a site in dev mode like this.

Reply:
This is the advanced section. I'm adding a link to the custom UI article for those who find it unfamiliar.